cpp/src/Slice/Preprocessor.h

```diff
@@ -43,25 +43,16 @@ namespace Slice
         };
 
         bool printMakefileDependencies(
-            std::ostream&,
-            Language,
-            const std::vector<std::string>&,
-            const std::string& = "",
-            const std::string& = "cpp",
-            const std::string& = "");
-        bool printMakefileDependencies(
-            std::ostream&,
-            Language,
-            const std::vector<std::string>&,
-            const std::vector<std::string>&,
-            const std::string& = "cpp",
-            const std::string& = "");
```

Comment:
This overload was only called by `slice2java`, seemingly by accident though.
After fixing Java, this was dead code.